From f47ccb3ea858baf1269b384073f28a46f71b5698 Mon Sep 17 00:00:00 2001
From: Ileana Dumitrescu <ileanadumitrescu95@gmail.com>
Date: Tue, 20 May 2025 19:48:10 +0300
Subject: [PATCH] gnulib: Recognize MSVC linker option

The '-def' option is used to specify a module-definition file
to the linker in MSVC.

* gnulib/build-aux/compile: Add '-def' as a linker option.
---
 gl/build-aux/compile.diff | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 gl/build-aux/compile.diff

diff --git a/gl/build-aux/compile.diff b/gl/build-aux/compile.diff
new file mode 100644
index 00000000..8bd59326
--- /dev/null
+++ b/gl/build-aux/compile.diff
@@ -0,0 +1,12 @@
+--- gnulib/build-aux/compile
++++ gl/build-aux/compile
+@@ -139,6 +139,9 @@ func_cl_wrapper ()
+       eat=
+     else
+       case $1 in
++	-def*)
++          linker_opts="$linker_opts $1"
++	  ;;
+ 	-o)
+ 	  # configure might choose to run compile as 'compile cc -o foo foo.c'.
+ 	  eat=1
-- 
2.45.2

